<ui version="4.0" >
 <class>dlgSaveShapeFiles</class>
 <widget class="QDialog" name="dlgSaveShapeFiles" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>204</height>
   </rect>
  </property>
  <property name="font" >
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle" >
   <string>Save SHAPE Files</string>
  </property>
  <property name="modal" >
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>160</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkSavePoints" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save Point SHAPE File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGetPointFilename" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>50</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Set Points SHAPE Filename . . .</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGetLineFilename" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>120</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Set Lines SHAPE Filename . . .</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkSaveLines" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>90</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Save Lines SHAPE File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnePointsFileName" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>50</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lneLinesFileName" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>240</x>
     <y>120</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgSaveShapeFiles</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgSaveShapeFiles</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
